packages/jellyfish-core/__tests__/container_adapter_client.ts

```diff
@@ -29,8 +29,8 @@ export class ContainerAdapterClient extends JellyfishClient {
 
     const { result, error } = response
 
-    if (error !== null) {
-      throw new JellyfishError(error)
+    if (error !== undefined && error !== null) {
```

Comment:
is error != null better here? so it captures both undefined and null

Reply:
In JavaScript `undefined !== null` as the linting tool was complaining too, `null` works for that context but I added `undefined` to be extra sure. Usually people do this: `if (error) {` but the linting tool didn't like that either 😥.